gravitee-apim-gateway/gravitee-apim-gateway-coverage/pom.xml

```diff
@@ -205,6 +205,12 @@
             <artifactId>gravitee-apim-gateway-tests-sdk</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>io.gravitee.apim.gateway</groupId>
+            <artifactId>gravitee-apim-gateway-tests-integration</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Why do we want to add this module to the test coverage ? Are you testing the test sdk with unit test ?